site/collections/_projects/room-for-change.md

```diff
@@ -28,14 +28,9 @@ The impact was measurable: from 2022 to 2023 the number of participants who have
 <b>Project Status:</b> Completed  <br />
 <b>Timeline:</b> 4 months (1-10-2023 - 31-1-2024)  <br />
 <b>Location:</b> Remote<br />
-<b>Skills:</b> Business Intelligence, Data Engineering, Google toolbox [(Looker)](https://cloud.google.com/looker) <br />
-<b>Team:</b> 4 Data Scientists (Correlaid), 2 Domain Experts (ROOM FOR CHANGE), 1 Project Coordinator (Correlaid), 1 Project Planner (Correlaid) <br />
+<b>Topic:</b> Streamlined Data Management & Dashboarding for Temporary Housing Impact Monitoring <br />
+<b>Topics:</b> Data management / data cleaning / automated reporting / dashboarding
```

Comment:
I would argue to not add the team, for several reasons: 
* We don't have this information on all projects, making it hard to remain consistent over all the project pages
* The intended audience of the project page will not be particularly interested in the details of the team makeup. The number of volunteers is clear from the list of names, their exact titles and roles are relatively irrelevant, as we mention what the project entails. 

I feel like there's maybe even more information that we can cut from this section. This information was in the original call for volunteers, because it's relevant when explaining the specific project to new people, but it's a little bit too much detail and not specifically relevant for people who just want to get a sense of what kinds of projects CorrelAid does.